packages/indexer-service/package.json

```diff
@@ -28,7 +28,8 @@
     "@connext/vector-utils": "0.2.1",
     "@google-cloud/profiler": "4.1.1",
     "@graphprotocol/common-ts": "1.3.3",
-    "@graphprotocol/indexer-common": "^0.12.0",
+    "@graphprotocol/indexer-common": "^0.13.0",
+    "@graphprotocol/indexer-native": "file:../indexer-native",
```

Comment:
I'm going to fix this one before we merge, but let's first address the review.

Reply:
Seems there are two issues here. I think you're referring to the local path. Another issue is now that when I run yarn it says it can't find version 0.13.0 for indexer-common.